CreateNotificationActivity/handler.ts

```diff
@@ -154,8 +154,12 @@ export const getCreateNotificationActivityHandler = (
 
   // whether email notifications are enabled in this user profile - this is
   // true by default, it's false only for users that have isEmailEnabled = false
-  // in their profile
-  const isEmailEnabledInProfile = profile.isEmailEnabled;
+  // in their profile. We assume it's true when not defined in user's profile.
+  const isEmailEnabledInProfile = profile.isEmailEnabled !== false;
+
+  // Check if the email in the user profile is validated.
+  // we assume it's true when not defined in user's profile.
+  const isEmailValidatedInProfile = profile.isEmailValidated !== false;
```

Comment:
@gunzip Could you confirm the field `isEmailValidated` is set to **false** on CIE first authentication? 
Because this assumption is valid only for SPID users

Reply:
yes, see https://github.com/pagopa/io-backend/blob/master/src/controllers/authenticationController.ts#L95

Reply:
this fix regards existing (SPID) users that never had this field set up (so it's undefined in their profile)